lib/workers/repository/process/lookup/index.ts

```diff
@@ -254,11 +254,26 @@ export async function lookupUpdates(
           return res;
         }
         const newVersion = release.version;
+        const fromVersion = lockedVersion || currentVersion;
+        if (!fromVersion) {
+          logger.debug(
+            {
+              currentDigest,
+              currentValue,
+              datasource,
+              depName,
+              lockedVersion,
+              packageFile,
+            },
+            'No dep currentVersion or lockedVersion'
+          );
+          continue;
```

Comment:
Will this skip docker digest updates? There `currentVersion` can be undefined when implicit latest tag is used.

Reply:
The problem occurs when `getUpdateType(config, versioning, currentVersion, newVersion)` is called and `currentVersion` is null. I'm not 100% sure of the best place to do this fix.

Reply:
So we should at least do a dry run against a repo with docker dicgest updates to validate

Reply:
Tried another approach